README.md

```diff
@@ -6,37 +6,120 @@
+
+
+### Adding DICOM Viewer to the oCIS Deployment Example
+oCIS provides some [deployment examples](https://github.com/owncloud/ocis/tree/master/deployments/examples/) including detailed configuration step by step guides for [local production setup](https://doc.owncloud.com/ocis/next/depl-examples/ubuntu-compose/ubuntu-compose-prod.html) and [deployment of Infinite Scale on the Hetzner Cloud](https://doc.owncloud.com/ocis/next/depl-examples/ubuntu-compose/ubuntu-compose-hetzner.html).
+In both cases, it only takes a few very small and simple steps to add the DICOM Viewer Web Extension to the [`ocis_full` deployment example](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/) of your own installation:
+
+1. Navigate to the `/opt/compose/ocis/ocis_full` folder of your installation and copy [`dicom-viewer.yml`](https://github.com/owncloud/web-app-dicom-viewer/blob/main/dicom-viewer.yml) into the [`web_extensions`](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/web_extensions) subfolder.
+
+2. Add `DICOMVIEWER=:web_extensions/dicom-viewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `/opt/compose/ocis/ocis_full`).\
+Your `.env` file should now look like this:
+   ```
+   ## oCIS Web Extensions ##
+   # It is possible to use the oCIS Web Extensions to add custom functionality to the oCIS frontend.
+   # For more details see https://github.com/owncloud/web-extensions/blob/main/README.md
+   
+   <list of all web extensions>
+   
+   DICOMVIEWER=:web_extensions/dicom-viewer.yml
+   ```
+
+3. Append `${DICOMVIEWER:-}` to the `COMPOSE_FILE` variable at the very end of the last line of the `.env` file. This variable combines the configs of all the components that need to be loaded.
+
+   `COMPOSE_FILE=docker-compose.yml${OCIS:-} ... <variables of lots of other configs that are added to docker compose> ... ${DICOMVIEWER:-}`
+
+   After appending `${DICOMVIEWER:-}`, your `.env` file should look like this:
+   ```
+   ## IMPORTANT ##
+   # This MUST be the last line as it assembles the supplemental compose files to be used.
+   # ALL supplemental configs must be added here, whether commented or not.
+   # Each var must either be empty or contain :path/file.yml
+   COMPOSE_FILE=docker-compose.yml${OCIS:-}${TIKA:-}${S3NG:-}${S3NG_MINIO:-}${COLLABORA:-}${MONITORING:-}${IMPORTER:-}${CLAMAV:-}${ONLYOFFICE:-}${INBUCKET:-}${EXTENSIONS:-}${UNZIP:-}${DRAWIO:-}${JSONVIEWER:-}${PROGRESSBARS:-}${EXTERNALSITES:-}${DICOMVIEWER:-}
+   ```
+
+4. Update the `csp.yaml` file located in `/config/ocis/csp.yaml` according to the description in https://github.com/owncloud/awesome-ocis/blob/main/webApps/apps.json#L374 
```

Comment:
```suggestion
4. Update the `csp.yaml` file located in `/config/ocis/csp.yaml` by adding the 
`"id": "com.github.owncloud.web-app-dicom-viewer"` block as shown in the [webApps/apps.json](https://github.com/owncloud/awesome-ocis/blob/main/webApps/apps.json) example.
```
This suggestion makes the block more clear and does btw not reference any line number which might change

Reply:
Thanks for the suggestion. Yes this is much better as line numbers might change when more apps are added to that file